src/source_map_utils.ts

```diff
@@ -97,16 +97,12 @@ export function sourceMapGeneratorToConsumer(
   return new SourceMapConsumer(rawSourceMap);
 }
 
-export function sourceMapTextToConsumer(sourceMapText: string): SourceMapConsumer {
-  // tslint:disable-next-line:no-any constructor actually supports text.
-  const sourceMapJson: any = sourceMapText;
-  return new SourceMapConsumer(sourceMapJson);
+export function sourceMapTextToConsumer(sourceMapText: string): BasicSourceMapConsumer {
+  return new SourceMapConsumer(sourceMapText) as BasicSourceMapConsumer;
```

Comment:
It turns out it isn't - does me changing the signature to BasicSourceMapConsumer | IndexedSourceMapConsumer (matching what's returned by the constructor) alleviate your concern?